pkg/handlers/apitests.go

```diff
@@ -46,27 +67,16 @@ func NewBaseHandlerTestSuite(sender notifications.NotificationSender, packageNam
 // HandlerConfig interface, so overrides can be made to the config
 func (suite *BaseHandlerTestSuite) HandlerConfig() *Config {
 	return &Config{
-		db:     suite.DB(),
-		logger: suite.Logger(),
+		db:              suite.DB(),
+		logger:          suite.Logger(),
+		appNames:        ApplicationTestServername(),
+		sessionManagers: setupSessionManagers(),
 	}
 }
 
-func (suite *BaseHandlerTestSuite) SetupSessionManagers() [3]*scs.SessionManager {
-	var milSession, adminSession, officeSession *scs.SessionManager
-	store := memstore.New()
-	milSession = scs.New()
-	milSession.Store = store
-	milSession.Cookie.Name = "mil_session_token"
-
-	adminSession = scs.New()
-	adminSession.Store = store
-	adminSession.Cookie.Name = "admin_session_token"
-
-	officeSession = scs.New()
-	officeSession.Store = store
-	officeSession.Cookie.Name = "office_session_token"
-
-	return [3]*scs.SessionManager{milSession, adminSession, officeSession}
+func setupSessionManagers() auth.AppSessionManagers {
+	return auth.SetupSessionManagers(nil, false, time.Duration(180*time.Second),
+		time.Duration(180*time.Second))
```

Comment:
extreme nitpick: I think wrapping with `time.Duration(...)` here is redundant.
```suggestion
		180*time.Second
```

Reply:
Thank you!